defaults/src/main/resources/default.properties

```diff
@@ -547,8 +547,6 @@ transfer.connection.retry.delay=0
 connection.hostname.default=
 # Convert hostname to Punycode
 connection.hostname.idn=true
-# java.net.preferIPv6Addresses
-connection.dns.ipv6=false
```

Comment:
We should probably only just change this default to `true`.

Reply:
I'd really rather that the OS handles this (instead of implementing application-level hacks that shouldn't be there in the first place).

Reply:
I would prefer too, but this also depends on defaults in the Java Runtime.

Reply:
Even Sun Java supported this back in 2002: https://docs.oracle.com/javase/7/docs/technotes/guides/net/ipv6_guide/index.html - if there's a JRE out there that has no IPv6 support two decades later - it can only be described as: objectively garbage.

We should merge this, get a beta build out that users can heavily test.

Reply:
Sure the IPv6 stack is available but we are just talking about order preferences in name resolution always prioritizing IPv4.

> By default, we would prefer IPv4 addresses over IPv6 addresses, i.e., when querying the name service [^1] 

[^1]: https://docs.oracle.com/javase/7/docs/technotes/guides/net/ipv6_guide/index.html#ipv6-related